pyproject.toml

```diff
@@ -96,6 +96,7 @@ anthropic = "^0.44.0"
 google-generativeai = "^0.8.4"
 js2py = "^0.74"
 retry2 = "^0.9.5"
+docstring-parser = "^0.16"
```

Comment:
isn't that just for dev purposes?

Reply:
It will be used by the script we'll run in tests. It will validate that snippets are rendered and match the source code.

Reply:
Otherwise this whole thing will degrade...

Reply:
it can still be just under dev group. no need to include that in the docker image, for example

Reply:
got it! <3